ds3_java_cli/src/main/java/com/spectralogic/ds3cli/Arguments.java

```diff
@@ -494,6 +495,15 @@ public void printHelp() {
         helpFormatter.printHelp("ds3", this.options);
     }
 
+    // take an enum.values() and print all possible values
+    private static String printEnumOptions(Object[] optionEnum) {
+        String ret = "";
```

Comment:
I think you should use a StringBuilder instead of a String, because it is much faster and consumes less memory.